### ISSUE TITLE
Unable to save model

### ISSUE THREAD

**Issue by ukemi** [geneontology/noctua-form#165]
Several of us have tried to save 
http://noctua.geneontology.org/editor/graph/gomodel:6205c24300000538?

When we do, it looks like it saves and then reverts back to unsaved.

**Comment by vanaukenk** [comment]
From 2022-03-18 call:

We will continue to keep an eye on this behavior.  If it happens again, we'll try to get everyone to look at it together.

**Comment by ukemi** [comment]
http://noctua-dev.berkeleybop.org/workbench/noctua-form/?model_id=gomodel%3A6205c24300000538

**Comment by hdrabkin** [comment]
FWIW, today, changing a Dev to Prod, I had to do it multiple times before the landing page finally showed I had changed it.  Each time I went back to the pull-down to change, it had 'Development' rather than Prod, even after changing and then saving.  I noticed that the page did not seem to 'blink' after a save. However, the save that worked 'blinked' after I tried again and that was the one that seemed to work.
